tox.ini

```diff
@@ -30,5 +30,5 @@ deps =
     black
     isort
 commands =
-    black tests scenario runtime
-    isort --profile black tests scenario runtime
+    black tests scenario scenario/runtime
```

Comment:
it should just be 'scenario'. Forgot to change it after I moved runtime inside scenario/, it used to be two folders on the same level.